plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java

```diff
@@ -54,7 +57,13 @@ public void init_test_from_accepted_sources() throws Exception {
         // should be picked up
         Path target = Paths.get("jenkins.tmp");
         Path newLink = Paths.get(tempFolder.getRoot().getAbsolutePath(), "jenkins_5.yaml");
-        Files.createSymbolicLink(newLink, target);
+
+        try {
+            Files.createSymbolicLink(newLink, target);
+        } catch (IOException e) {
+            // often fails on windows due to non admin users not having symlink permission by default, see: https://stackoverflow.com/questions/23217460/how-to-create-soft-symbolic-link-using-java-nio-files/24353758#24353758
+            Assume.assumeFalse(Functions.isWindows());
```

Comment:
This should be enough?
```suggestion
            Assume.assumeFalse(Functions.isWindows());
            throw e;
```

Reply:
Committed :+1: